tests/test_core.py

```diff
@@ -4039,7 +4050,8 @@ def test_dylink_hyper_dupe(self):
         only_in_second_0();
         only_in_third_0();
       }
-    ''' % dylib_suffix, side=r'''
+    ''' % dylib_suffix,
+                     side=r'''
```

Comment:
I don't care much, but the new indentation looks odder to me... it's aligned with the `main=` from above I suppose.